label_studio_converter/__init__.py

```diff
@@ -5,4 +5,4 @@
     print(e)
 
 
-__version__ = '0.0.46.dev'
+__version__ = '0.0.47.rc0'
```

Comment:
Don't change the version - it should be updated automatically with CI run
cc @farioas